tests/languages/ruby/tests.rb

```diff
@@ -45,3 +45,8 @@
 
 response = general.redirect();
 puts response['result']
+
+
+file = Appwrite::AppwriteFile.new('./tests/resources/file.png')
```

Comment:
Are we sure we can't just use Appwrite::File.new? I also notice we still use `File.new()` for the code examples.

Reply:
I am sure, as inside the Appwrite module itself we will be using File and that will clash with the File keyword.

Reply:
I was unable to make it work locally I tried various options.

Reply:
@eldadfux Found a way to make it as you requested, I have updated to make it (`Appwrite::File.new(...)`) to work.

Reply:
Nice! Restarted the build to make sure tests pass.